foundations/javascript_basics/fundamentals-4.md

```diff
@@ -5,49 +5,47 @@
+In many ways, TDD is much more productive than writing code without tests. If we didn't have the test for the adding function above, we would have to run the code ourselves over and over, plugging in different numbers until we were sure that it was working... not a big deal for a simple `add(2, 2)`, but imagine having to do that for more complicated functions, like checking whether or not someone has won a game of tic tac toe: \(`game_win(["o", null,"x",null,"x",null,"x", "o", "o"])`). If you didn't do TDD, then you might actually have to play multiple games against yourself just to test if the function was working correctly!
 
-We will teach you the art of actually writing these tests later in the course.  The following exercises have the tests already written out for you. All you have to do is read the specs and write the code that makes them pass!  The very first exercise \(`01-helloWorld`\) is intentionally very simple and walks you through the process of running the tests and making them pass.  
+We will teach you the art of actually writing these tests later in the course. The following assignment has the tests already written out for you. All you have to do is set up the testing environment, read the specs, and write the code that makes them pass!
 
-### Good Luck!
+### Assignment
 
-Check out our exercises repository [here](https://github.com/TheOdinProject/javascript-exercises) and follow the directions in the [README](https://github.com/TheOdinProject/javascript-exercises#how-to-use-these-exercises) for setting up Jest.  Solutions for the exercises can be found on the 'solutions' branch of that repo. 
+<div class="lesson-content__panel" markdown="1">
+Follow the steps below to get started. Once you complete Step 1, be sure to use the README for **_each individual exercise_** in order to do them correctly.
 
-**NOTE:** You must read the README for the repository and each individual exercise in order to do them correctly.  
-
-Complete the following exercises:
-
-- helloWorld
-- repeatString
-- reverseString
-- removeFromArray
-- sumAll
-- leapYears
-- tempConversion
+1. Set up the files and Jest by carefully following the directions in the [repository's README](https://github.com/TheOdinProject/javascript-exercises#readme).
+2. Now that you have cloned the repository and run `npm install` to install Jest, complete these exercises in the following order:
+   - helloWorld (This exercise is intentionally very simple to ensure that you have set up everything properly!)
+   - repeatString
+   - reverseString
+   - removeFromArray
+   - sumAll
+   - leapYears
+   - tempConversion
+3. See how your solution compares to the exercise's given solution. Solutions for the exercises can be found on the 'solutions' branch of the repo.
```

Comment:
Really love this addition. Can we reword it to put a little more emphasis to have a "working solution" instead of just a "solution", or something along the lines of "a solution that passes all the tests"? 

I ask this because I believe there's value in being as explicit as possible here, so that users only look at the given solutions once they have a solution that works for them.

I have a suggestion here, but feel free to change the wording if it makes the message better.

```suggestion
3. Once you have a working solution, see how it compares to the exercise's given solution. Solutions for the exercises can be found on the 'solutions' branch of the repo.
```

Reply:
I thought the same thing but for whatever reason didn't include it! I've tried inserting an emphasis that the working solution should pass all the tests, but it kept feeling/sounding too verbose. I've opted instead to use your suggestion.